Library/Homebrew/cmd/audit.rb

```diff
@@ -1234,4 +1224,9 @@ def audit_urls
   def problem(text)
     @problems << text
   end
+
+  if ARGV.switch? "D"
+    Homebrew.inject_dump_stats!(FormulaAuditor, /^audit_*/)
+  end
```

Comment:
This still needs to be moved into `def audit`. We don't want this code already to run for `require "cmd/audit"`.